stock_management/tmp_win_utils.py

```diff
@@ -0,0 +1,36 @@
+from stock_management.layouts import get_test_layout
+import PySimpleGUI as sg
+import time
+
+
+def test_session(
+    test_events=[],
```

Comment:
Notice that I added here some arguments, in order to conform to the other session function input signature (see [here](https://github.com/francescocarpanese/stock_management/blob/test_stefano/stock_management/drugs_win_utils.py#L87) for example )

This , together with the last part of the lines, allows you to create the session and trigger some event externally from the function. This is used in the test suite to be able to automatically trigger some events on this windows and check the results.